src/SDLRenderingWindow.cpp

```diff
@@ -143,8 +143,9 @@ void SDLRenderingWindow::NextDisplay()
         SDL_Rect bounds;
         SDL_GetDisplayBounds(display, &bounds);
 
-        poco_debug_f(_logger, "Bounds for display %?d: X=%?d Y=%?d W=%?d H=%?d",
-                      display, bounds.x, bounds.y, bounds.w, bounds.h);
+        std::stringstream debugOutput;
```

Comment:
Looks like I forgot to remove this variable when changing from using stringstream to just calling poco_debug twice. Is there some simple way to change it from here? Otherwise maybe just remove this line when you can :^)